environments/64-test/public.yml

```diff
@@ -0,0 +1,177 @@
+# For a real deployment
+# should be e.g. 'www.commcarehq.org'
+internal_domain_name: 'localdomain'
+
+etc_hosts_lines: []
+etc_hosts_lines_removed: []
+
+touchforms_enabled: false
+
+riak_backend: "bitcask"
+riak_ring_size: 64
+
+ansible_user_password_sha_512: "{{ secrets.ANSIBLE_USER_PASSWORD_SHA_512 }}"
+
+dev_users:
+  present:
+    - dmiller
+    - droberts
+    - jwilson
+    - npellegrino
+  absent: []
+
+# use this value to switch between nginx and apache
+proxy_type: nginx
+
+elasticsearch_endpoint: '{{ groups.elasticsearch.0 }}'
+elasticsearch_memory: '128m'
+elasticsearch_cluster_name: 'deves'
+
+# uncomment to only allow access to eth1
+#ufw_private_interface: eth1
+
+# uncomment and change to the control machine IP when using ufw
+# this ensures that ssh access from the control machine is always allowed regardless of other firewall rules
+#control_machine_ip: 127.0.0.1
+
+ECRYPTFS_PASSWORD: "{{ secrets.ECRYPTFS_PASSWORD }}"
+
+KSPLICE_ACTIVE: no
+KSPLICE_ACTIVATION_KEY: "{{ secrets.KSPLICE_ACTIVATION_KEY }}"
+
+DATADOG_ENABLED: True
+DATADOG_API_KEY: "{{ secrets.DATADOG_API_KEY }}"
+DATADOG_APP_KEY: "{{ secrets.DATADOG_API_KEY }}"
+
+CLOUDANT_CLUSTER_NAME: null
+
+
+# This dumps the dummy keystore, but allows the ansible script to continue, should be overridden for real deploys
+#keystore_file: 'DimagiKeyStore'
+
+backup_es: False
+
+postgres_users:
+  commcare:
+    username: commcarehq
+    password: commcarehq
```

Comment:
any reason not to use strong passwords?

Reply:
Thanks, good catch. I just moved this config for all the "example" environments (development, travis, bootstrap) into the private/vault file, and changed the password to something "passwordy" so its easier to find and replace with a real password. (Longer term we're probably going to want to be auto-generating these passwords for a new env, but this smaller change puts is more in line with the current convention.)

For this current env (64-test), we're just going to be doing some loadtesting and then probably deleting the actual machine. For just that, I'm fine leaving the weak password here, especially because of how strict postgresql IP whitelisting is (so even if there's some terrible network misconfiguration, someone will have to ssh into the machine before being able to access postgresql). If we reuse the config behind this env, I'll update vault to have a real password for this.